documentation/source/overview/concepts.rst

```diff
@@ -236,6 +155,8 @@ look at:
 https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir
 
 
+
+
 Matlab Integration on Mac
```

Comment:
I am wondering if `Matlab Integration for Mac` might be better suited for the `Installation` section. To me, it doesn't seem to fit with the other "SCT Concepts".

Reply:
Side question: should I be providing suggestions on content like this? Or is it preferred to simply comment only on the changes made by you?

Reply:
> Side question: should I be providing suggestions on content like this? Or is it preferred to simply comment only on the changes made by you?

you are more than welcome to provide suggestions!

Reply:
> I am wondering if `Matlab Integration for Mac` might be better suited for the `Installation` section. To me, it doesn't seem to fit with the other "SCT Concepts".

fixed in 3f095f8c82c0198528b876287a5b57eb3d37c7f7